docs/source/guide/Modules.lrad

```diff
@@ -0,0 +1,107 @@
+# Modules
+
+To create a module, use the `module` special form. All modules must start with a
+*module declaration* which provides the module with a name, documentation string
+and an exports vector. Module declarations are evaluated, so you must quote any exports.
+
+Here is a simple example of a module:
+
+    (module
+      {:module  'my-module
+       :doc     "Just for showing off modules."
+       :exports '[foo bar]}
+
+      (def foo-helper
+        (fn [x] (+ x 1)))
+
+      (def foo
+        (fn [x] (foo-helper (foo-helper x))))
+
+      (def bar
+        (fn [x] (foo (+ x 2)))))
+
+To import a module, use `import`:
+
+    (import my-module)
+
+The definitions inside a module are evaluated in the current scope, so you can
```

Comment:
Sorry to be picky but I think it’s important to be precise here to avoid confusion: Can we change this to

> The definitions inside a module are evaluated in the scope of the module definition site.

It would also be helpful to have another example like this

~~~clojure
(def foo :foo)
(module
  { ... }
  (def bar foo))